guidelines/releases/release-process.md

```diff
@@ -13,22 +13,18 @@ To prepare a release, define new and remove old release
 Define release jobs on the `master` branch in the `test-infra` repository. To ensure every job name is unique, prefix it with `pre-rel{versionNumber}`. Remember to provide the version number without any periods. To learn how to define a release job for a component, read the following [document][2]. To find all jobs for the 0.9 release, look for job names with the `pre-rel09` prefix.
```

Comment:
Move the last sentence as an example (one sentence earlier). Also: finish the sentence in the previous paragraph.